mocha/common.js

```diff
@@ -0,0 +1,8 @@
+global.React = require('react');
+global.sinon = require('sinon');
+global.chai = require("chai");
+global.should = chai.should();
```

Comment:
Should puts it's grubby fingers over all the objects in your app... not awesome.